active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,572 @@
+    </muxc:TeachingTip>
+</muxc:TeachingTip.Attach>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+Here's the result when this code runs and the clicks the button.
+
+<a href="url"><img src="TeachingTipSampleApp.jpg" height="600"></a>
+
+### Add a margin 
+
+A teaching tip can have a margin added by setting the Margin property. When a teaching tip is attached to a UI element, a margin will move it away from the UI element it is pointing to. When a teaching tip is not attached to anything, a margin will move it away from the side of the app window it is positioned towards. 
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick"></Button>
+<muxc:TeachingTip> 
+    <muxc:TeachingTip x:Name="AutoSaveTip"
+        Title="Saving automatically"
+        Subtitle="We save your changes as you go - so you never have to."
+        Margin="80">
+    </muxc:TeachingTip>
+</muxc:TeachingTip.Attach>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipMarginSample.jpg" height="600"></a>
+
+### Point to a UI element
+
+You can change the teaching tip to point to a specific UI element by setting the TeachingTipPlacementMode property. For a targeted teaching tip, this placement will be relative to the UI element target. For a non-targeted teaching tip, this placement will relative to app window.  
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipPointerSample.jpg" height="600"></a>
+
+### Add content
+
+Content can be added to a teaching tip using the Content property. If there is more content to show than what the size of a teaching tip will allow, a scrollbar will be automatically enabled to allow a user to scroll the content area. 
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="15">
+                <CheckBox Content="Don't show tips at start up" Checked="HandleChecked" Unchecked="HandleUnchecked" />
+                <p>You can change your tip preferences in <a href="app:/item/SettingsPage">Settings</a> if you change your mind.</p>
```

Comment:
HTML content isn't supported here.  I think this was meant to be a TextBlock, and the two wrapped in a StackPanel.

Reply:
```suggestion
                <TextBlock>You can change your tip preferences in <a href="app:/item/SettingsPage">Settings</a> if you change your mind.</TextBlock>
```